_includes/footer.html

```diff
@@ -199,15 +200,189 @@
   });
   createProgressBar(section, implementationPoints, maxPoints, stubPenalties, missing, stubs);
 }
+async function makeD3Chart(data) {
```

Comment:
I don't think making it `async` changes anything, as there are no `await`s here?

Reply:
Fixed this and a few other functions. But somehow my commits got messed up. I'm not even sure squash on merge will work for this, but if it does that would be best. However, I suspect it will complain about merge conflicts.